code/__DEFINES/dcs/signals.dm

```diff
@@ -477,6 +477,13 @@
 #define COMSIG_MACHINERY_POWER_LOST "machinery_power_lost"
 ///from base power_change() when power is restored
 #define COMSIG_MACHINERY_POWER_RESTORED "machinery_power_restored"
+///from /obj/machinery/set_occupant(atom/movable/O): ()
```

Comment:
```suggestion
///from /obj/machinery/set_occupant(atom/movable/O): (new_occupant)
```

Reply:
I thought `: ()` indicated the return value here?